uber/automated_emails_server.py

```diff
@@ -15,7 +15,8 @@ class AutomatedEmail:
     queries = {
         Attendee: lambda session: session.all_attendees().options(
             subqueryload(Attendee.admin_account)).options(
-            subqueryload(Attendee.dept_checklist_items)),
+            subqueryload(Attendee.dept_checklist_items)).options(
+            subqueryload(Attendee.group)),
```

Comment:
Eagerly load the `Attendee.group` relationship to prevent thousands of individual SQL queries to pull up the attendee's group record.